lib/core/player.rb

```diff
@@ -6,12 +6,21 @@ def initialize(symbol)
   end
 
   def make_move(board)
-    move = get_move(board)
-    board.check_tile(move, self)
+    board.mark_tile(get_move(board), self)
   end
 
   def get_move(board)
     raise NotImplementedError.
             new("#{self.class.name}# has not implemented the method get_move")
   end
+
+  def ==(other)
+    return false if other.nil?
+    @symbol == other.symbol
+  end
+
+  def !=(other)
```

Comment:
Did you have a test that showed you needed to do this? I have a hunch ruby is going to use the overridden == method so this may be duplicated. I'd let a test drive that though.